tests/runtest.sh

```diff
@@ -4,7 +4,7 @@ set -e
 PREFIX="`hostname`-`id --user`-`echo $$`-"
```

Comment:
I'd document and validate the interface like this:

# $1 = path to generated test binary
...
if [ $# != 1 ]; then
fi